blaze-core/src/test/scala/org/http4s/blaze/TestHead.scala

```diff
@@ -35,6 +35,7 @@ abstract class TestHead(val name: String) extends HeadStage[ByteBuffer] {
     closed = true
     super.stageShutdown()
     p.trySuccess(ByteBuffer.wrap(getBytes()))
+    ()
```

Comment:
That said, `Future[Unit]` silently treated as `Unit` was a big pain in my Akka days. I think this is a valuable flag on balance.  But it is the one that left the most curious artifacts in this PR.